webapp/app/config.py

```diff
@@ -95,6 +95,12 @@ class DevelopmentConfig(BaseConfig):
     SESSION_COOKIE_SECURE = False  # Because Safari can not send Secure Cookies via HTTP to localhost
 
     ERICA_BASE_URL = environ.get('ERICA_BASE_URL') or 'http://0.0.0.0:8000/01'
+    """ Use the following variables to make tests locally with the job queue dummy routes.
+    USE_MOCK_API = True
+    MOCK_MIN_REQUEST_COUNT = 5
+    MOCK_DELIVER_FAIL_ON_POST = False
+    MOCK_DELIVER_FAIL_ON_GET = False
```

Comment:
Personally, I think those should not be configuration values that we set for the application but rather attributes of the MockErica similar to `MockErica.eric_process_not_successful_error_occurred` because we probably want to change these during runtime (e.g. set them differently for different tests) and this is also an implementation detail that we do not need to set here.